src/bug-filing/common/issue-details-getter.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { EnvironmentInfo } from '../../common/environment-info-provider';
+import { CreateIssueDetailsTextData } from '../../common/types/create-issue-details-text-data';
+import { IssueUrlCreationUtils } from './issue-filing-url-string-utils';
+export type IssueDetailsGetter = (
```

Comment:
What do you think of calling this `IssueDetailsFormatter` instead? Getter sounds more like retrieving, and one of these things is given the raw details and formats them.

Reply:
But is not a formatter. I though about provider or `buildIssueDetails`. In the end, I decided to just move it and not rename it.